qemu/tests/aio_test.py

```diff
@@ -0,0 +1,126 @@
+import functools
+import glob
+import logging
+import os
+import re
+
+from avocado import TestCancel
+from avocado import TestFail
+
+from avocado.utils import cpu
+from avocado.utils import process
+
+from virttest import data_dir
+
+
+def which(cmd):
+    """Check cmd if its executable is available in environment."""
+    logging.debug("Check if command '%s' is available", cmd)
+    paths = os.environ.get("PATH", "").split(":")
+    for path in paths:
+        cmd_path = os.path.join(path, cmd)
+        if os.path.isfile(cmd_path) and os.access(cmd_path, os.X_OK):
+            return cmd_path
+    raise TestCancel("No command %s available" % cmd)
+
+
+def coroutine(func):
+    """Start coroutine."""
+    @functools.wraps(func)
+    def start(*args, **kargs):
+        cr = func(*args, **kargs)
+        cr.send(None)
+        return cr
+    return start
+
+
+def get_qemu_version(target=None):
+    """Get installed QEMU version."""
+    logging.debug("check QEMU version")
+    line = process.run("qemu-img --version").stdout_text.splitlines()[0]
```

Comment:
The command line should be changed to "/usr/libexec/qemu-kvm --version"
Because the qemu-img version does not match the qemu-kvm version when performing upstream tests.
like this :ERROR: Command 'brew download-build -q --rpm qemu-kvm-4.*.*.src.rpm' failed.\nstdout: b'No such rpm: qemu-kvm-4.*.*.src.rpm\n'\nstderr: b''\nadditional_info: None

Reply:
I don't know if upstream QEMU is available using `brew` to download.

Reply:
Upstream QEMU will not be named qemu-kvm, so the qemu version will detect the version of the local host downloaded from brew.